src/modules/household/components/EditHouseholdMemberTable.tsx

```diff
@@ -199,18 +223,54 @@ const EditHouseholdMemberTable = ({
       },
       HOUSEHOLD_MEMBER_COLUMNS.assignedUnit(currentMembers),
       {
-        header: '',
-        key: 'action',
-        width: '1%',
+        ...BASE_ACTION_COLUMN_DEF,
         render: (hc: HouseholdClientFieldsFragment) => (
-          // No extra perm check is required, because this button only allows removing WIP Enrollments,
-          // which only requires Can Edit Enrollments, which is already required for this page
-          <RemoveFromHouseholdButton
-            currentDashboardEnrollmentId={currentDashboardEnrollmentId}
-            householdClient={hc}
-            onSuccess={refetchHousehold}
-            householdSize={currentMembers.length}
-            ariaLabel={`Remove ${clientBriefName(hc.client)}`}
+          <TableRowActions
+            record={hc}
+            recordName={clientBriefName(hc.client)}
+            MenuProps={{
+              MenuListProps: {
+                dense: false,
+              },
+            }}
+            menuActionConfigs={[
```

Comment:
Now that we don't have the `RemoveFromHouseholdButton` anymore, can we pass these as a `rowSecondaryActionConfigs` prop directly to the table?